behat/tests/bootstrap/v2/common/DebugTrait.php

```diff
@@ -77,12 +77,11 @@ public function debug($name)
         echo "$file";
     }
 
-
-    public function formatAssertResponse(
-        $expecting,
+    public function bespokeAssert(
```

Comment:
The name of this function could be more explicit about its intended use. To me, it seems to be narrowly focused on if a string appears in another string so something like `assertStringContainsString` could be clearer for future function callers.

Reply:
done